README.md

```diff
@@ -1,9 +1,9 @@
 
 # Customized built-in &lt;map&gt; element
 
-[![Build Status](https://travis-ci.org/prushforth/Web-Map-Custom-Element.svg?branch=master)](https://travis-ci.org/prushforth/Web-Map-Custom-Element)
+[![Build Status](https://travis-ci.org/Maps4HTML/Web-Map-Custom-Element.svg?branch=master)](https://travis-ci.org/prushforth/Web-Map-Custom-Element)
 
-The customized built-in `<map>` element is a prototype [implementation](http://maps4html.github.io/Web-Map-Custom-Element/)
+The customized built-in `<map>` element is a prototype [implementation](https://travis-ci.org/github/Maps4HTML/Web-Map-Custom-Element)
```

Comment:
@prushforth was the change of link destination here intended?

Reply:
No it was a mistake. Thanks for picking that up. Will fix.